src/modules/LanItems.rb

```diff
@@ -198,108 +198,24 @@ def main
-        "iwl6000"        => "kernel-firmware",
-        "b43"            => "b43-fwcutter",
-        "b43-pci-bridge" => "b43-fwcutter",
-        "bcm43xx"        => "",
-        "prism54"        => "",
-        "spectrum_cs"    => "",
-        "zd1201"         => "",
-        "zd1211rw"       => "",
-        "acx"            => "",
-        "rt73usb"        => "",
-        "prism54usb"     => ""
-      }
-
       Yast.include self, "network/hardware.rb"
 
-      # the defaults here are what sysconfig defaults to
-      # (as opposed to what a new interface gets, in {#Select)}
-      @SysconfigDefaults = {
-        "BOOTPROTO"                    => "static",
-        "IPADDR"                       => "",
-        "PREFIXLEN"                    => "",
-        "REMOTE_IPADDR"                => "",
-        "NETMASK"                      => "",
-        "MTU"                          => "",
-        "ETHTOOL_OPTIONS"              => "",
-        "NAME"                         => "",
-        "STARTMODE"                    => "manual",
-        "IFPLUGD_PRIORITY"             => "0",
-        "WIRELESS_MODE"                => "Managed",
-        "WIRELESS_ESSID"               => "",
-        "WIRELESS_NWID"                => "",
-        "WIRELESS_AUTH_MODE"           => "open",
-        "WIRELESS_WPA_PSK"             => "",
-        "WIRELESS_KEY_LENGTH"          => "128",
-        "WIRELESS_KEY"                 => "",
-        "WIRELESS_KEY_0"               => "",
-        "WIRELESS_KEY_1"               => "",
-        "WIRELESS_KEY_2"               => "",
-        "WIRELESS_KEY_3"               => "",
-        "WIRELESS_DEFAULT_KEY"         => "0",
-        "WIRELESS_NICK"                => "",
-        "WIRELESS_CHANNEL"             => "",
-        "WIRELESS_FREQUENCY"           => "",
-        "WIRELESS_BITRATE"             => "auto",
-        "WIRELESS_AP"                  => "",
-        "WIRELESS_POWER"               => "",
-        "WIRELESS_EAP_MODE"            => "",
-        "WIRELESS_WPA_IDENTITY"        => "",
-        "WIRELESS_WPA_PASSWORD"        => "",
-        "WIRELESS_WPA_ANONID"          => "",
-        "WIRELESS_CLIENT_CERT"         => "",
-        "WIRELESS_CLIENT_KEY"          => "",
-        "WIRELESS_CLIENT_KEY_PASSWORD" => "",
-        "WIRELESS_CA_CERT"             => "",
-        "WIRELESS_EAP_AUTH"            => "",
-        "WIRELESS_PEAP_VERSION"        => "",
-        "WIRELESS_AP_SCANMODE"         => "1",
-        # default options for bonding (bnc#404449)
-        "BONDING_MODULE_OPTS"          => "mode=active-backup miimon=100",
-        # defaults for tun/tap devices
-        "TUNNEL_SET_OWNER"             => "",
-        "TUNNEL_SET_GROUP"             => "",
-        # Infiniband
-        # default mode for IPoIB devices suggested in fate#315501
-        "IPOIB_MODE"                   => "connected"
-      }
+      yaml_dir = File.expand_path("../../data", __FILE__) + "/network"
```

Comment:
Ehm, no this is not way to go, it kills y2update and various loading location. Correct way is https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Directory.rb#L125

problem is also when you want experimentaly change only one yaml file in y2update so it should looke like

`@s390_defaults = YAML.load_file(Directory.find_data_path("network/s390_defaults.yml")) if Arch.s390`
and similar to others.